### PR TITLE
Fixing many small issues

### PR DESCRIPTION
**Pull request by DedrickEnc**
This pull request fix small bugs about stock : 

- Fix the UI problem on the ```bhStockEntryExit``` component

- Fix the filter issue on the ```findTransfer``` modal

- Fix the placeholder problem in the ```bhFindPatient``` component

- Set the default behavior on filtering transfer movement by the current depot

- Update the e2e test

### CODE REVIEW

**Comment by DedrickEnc** [comment]
@mbayopanda 
Can you review this pull request?

**Comment by DedrickEnc** [comment]
@mbayopanda 
Can you review this pull request?

File: client/src/modules/templates/bhFindPatient.tmpl.html [hunk 1]
```diff
@@ -31,7 +31,7 @@
         ng-model="$ctrl.idInput"
         ng-if="$ctrl.selected == $ctrl.options.findById"
         ng-keypress="$ctrl.onKeyPress($event)"
-        translate-attr="{ 'placeholder': '$ctrl.selected.placeholder' }"
+        translate-attr="{ 'placeholder': '{{$ctrl.selected.placeholder}}' }"
         ng-required="$ctrl.required">
```
Comment (mbayopanda) [inline client/src/modules/templates/bhFindPatient.tmpl.html:35]: 👍

**Review by mbayopanda — APPROVED** [review]
LGTM

**Pull request merged**